shell/app/modules/project/common/components/issue/issue-relation.tsx

```diff
@@ -59,24 +59,27 @@ export const IssueRelation = React.forwardRef((props: IProps, ref: any) => {
   const defaultIssueType = initTypeMap[issueType];
   const { getIssueRelation, addIssueRelation, deleteIssueRelation } = issueStore.effects;
 
+  const getList = React.useCallback(() => {
+    getIssueRelation({ id: issueDetail.id }).then((res) => {
+      setRelatingList(res[0]);
+      setRelatedList(res[1]);
+    });
+  }, [getIssueRelation, issueDetail]);
+
   React.useEffect(() => {
     if (!ref.current) {
       // eslint-disable-next-line no-param-reassign
       ref.current = { getList };
     }
-  });
+    return () => {
+      ref.current = undefined;
```

Comment:
why should set this ref to `undefined`?

Reply:
because the ref could save the last target